recipes/pyfilemanager/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "pyfilemanager" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyfilemanager-{{ version }}.tar.gz
+  sha256: da224c2f4658d0005a842804e116c7eada10b97bafd4b60d634467bf13b05424
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python <4.0
```

This should match the pin in run
```suggestion
    - python >=3.7,<4.0
```

---

Thank you for reviewing my recipe, @xhochy. I made this change, and updated the package to v1.1.0 (latest version on pypi). The hash is also updated.

---

I updated staged-recipes on my fork, and I see other recipes in there. Sorry if I shouldn't have done that. Perhaps this is the reason some of the checks are currently failing?